Add minutes for 24 Nov 2020 meeting

SidFaber opened the pull request:
Signed-off-by: Sid Faber <sid.faber@canonical.com>

SidFaber committed:
Add minutes for 24 Nov 2020 meeting

Signed-off-by: Sid Faber <sid.faber@canonical.com>

mikaelarguedas commented on meetings/2020_11_24/README.md:
```diff
@@ -0,0 +1,110 @@
+
+## Administrivia
+
+Following a brief discussion, it was decided to move new meeting minutes to the [ros-security/github repo](https://github.com/ros-security/community). Existing meeting minutes in the [ROS wiki](http://wiki.ros.org/ROS2/WorkingGroups/Security) will not be ported.
```
```suggestion
Following a brief discussion, it was decided to move new meeting minutes to the [`ros-security/community` Github repository](https://github.com/ros-security/community). Existing meeting minutes in the [ROS wiki](http://wiki.ros.org/ROS2/WorkingGroups/Security) will not be ported.
```

SidFaber replied:
Updated in a6e2e0e927a898d850106dbe5e7725534954701b

mikaelarguedas commented on meetings/2020_11_24/README.md:
```diff
@@ -0,0 +1,110 @@
+
+Marco suggested [the Robotics Middleware Framework (RMF)](https://github.com/osrf/rmf_demos) as a reference implementation for ROS security. This should be ready to run with ROS 2; they have already done some work with security as well.
+
+Marco also asked about revoking keys: there's a need to handle that within RMF should an individual robot in a fleet be physically compromised. @JaimeMartin provided [information on CRLs from eProsima](https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/auth_plugin/auth_plugin.html#generating-the-certificate-revocation-list-crl).
```
We should use an homogeneous way to refer to individuals in the notes (Github handles, first name, initials...).

SidFaber replied:
Agreed--I experimented with github handles, but they don't carry through to markdown. Plan on linking to github profiles in the "attendee" list, but then using first names throughout similar to the style used on the wiki site.

mikaelarguedas commented on meetings/2020_11_24/README.md:
```diff
@@ -0,0 +1,110 @@
+
+## G-Turtle goals
+
+Five open items could become part of our G-Turtle deliverables:
```
One thing mentioned in the last meetings and that I'd like us to look into is the ability of having configurable security levels per topic, following the ones supported by DDS-Security: NONE SIGN, ENCRYPT
Currently SROS2 is all or nothing, either all topics are encrypted or no security feature is used at all. This doesn't scale well (or makes sense) for real systems. Tracking ticket and past work on the topic available at https://github.com/ros2/sros2/issues/130

SidFaber replied:
Added to the "reference implementation" goal in a6e2e0e927a898d850106dbe5e7725534954701b--as I understood the conversation, we need a good implementation with some complexity before we can begin configuring topic-level security.

mikaelarguedas commented on meetings/2020_11_24/README.md:
```diff
@@ -0,0 +1,110 @@
+
+## Administrivia
+
+Following a brief discussion, it was decided to move new meeting minutes to the [ros-security/github repo](https://github.com/ros-security/community). Existing meeting minutes in the [ROS wiki](http://wiki.ros.org/ROS2/WorkingGroups/Security) will not be ported.
```
Porting the past notes and replacing the content of the wiki page to point to this github repo would be preferred. although I understand it's cumbersome work

SidFaber replied:
I'll leave a few breadcrumbs to cross-link the wiki and the github repo, but I think the effort to port the WG content would be better spent on updating sros2 documentation.

mikaelarguedas commented on meetings/2020_11_24/README.md:
```diff
@@ -0,0 +1,110 @@
+
+Marco suggested [the Robotics Middleware Framework (RMF)](https://github.com/osrf/rmf_demos) as a reference implementation for ROS security. This should be ready to run with ROS 2; they have already done some work with security as well.
+
+Marco also asked about revoking keys: there's a need to handle that within RMF should an individual robot in a fleet be physically compromised. @JaimeMartin provided [information on CRLs from eProsima](https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/auth_plugin/auth_plugin.html#generating-the-certificate-revocation-list-crl).
```
```suggestion
Marco also asked about revoking keys: there's a need to handle that within RMF should an individual robot in a fleet be physically compromised. Jaime provided [information on CRLs from eProsima](https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/auth_plugin/auth_plugin.html#generating-the-certificate-revocation-list-crl).
```

SidFaber replied:
Updated in a6e2e0e927a898d850106dbe5e7725534954701b

mikaelarguedas commented on meetings/2020_11_24/README.md:
```diff
@@ -0,0 +1,110 @@
+
+The scope of this issue is much wider than just security. Success depends upon buy-in from both the micro-ROS community and from Open Robotics.
+
+The agrees to continue to move the discussion forward to flesh out a design, but not to perform any work on the code at this time.
```
```suggestion
The WG agrees to continue to move the discussion forward to flesh out a design, but not to perform any work on the code at this time.
```

SidFaber replied:
Updated in a6e2e0e927a898d850106dbe5e7725534954701b

mikaelarguedas reviewed (commented):
thanks @SidFaber for putting these together

SidFaber committed:
Clarify G-Turtle simulation goal

Link the reference implementation to ros2/sros2#130. Also a few minor grammar updates.

Signed-off-by: Sid Faber <sid.faber@canonical.com>

SidFaber commented:
Thanks for the thorough read-though, @mikaelarguedas !

kyrofa reviewed (commented):
This looks good to me, but I'm curious why you made the meeting a directory with a README instead of just a dated md file. In case we have some other collatoral during a meeting? If so, can we at least rename README to something like `minutes.md`?

SidFaber commented:
Directories so we can attache things (presentations?) to the meetings. README just to reduce clicks--the minutes render as soon as you land in the directory.

kyrofa reviewed (approved):
Oh fair point, okay.
